### DISCUSSION TITLE
Testing the Authelia configuration: no such file or directory

### DISCUSSION THREAD

**Discussion by Techal62** [authelia/authelia#7598] [Q&A] [answered]
Hello,

#docker run authelia/authelia:latest authelia config validate --config /volume1/docker/authelia/config/configuration.yml

I have a problem getting my configuration file validation to work

![1](https://github.com/user-attachments/assets/747b70f8-a350-4457-ae02-59b1bff39d8b)

Excerpt from my docker-compose file

![2](https://github.com/user-attachments/assets/d3068c9f-552d-49fd-be32-a0667048cc47)

**Comment by james-d-elliott** [comment, marked as answer]
The `/volume1/docker/authelia/config/configuration.yml` path is not bound within the container, that's the path outside the container.

**Reply by Techal62** [reply]
Thanks for your help, but how can I test my config?
Because I run Authelia in a docker on a Synology NAS.
When I run the command docker run authelia/authelia:latest authelia config validate I am on the ssh console of my synology.
Do I need to enter the docker image to run the command?

**Reply by james-d-elliott** [reply]
It's the same as any container, you just need to bind the file or folder via a volume then use the absolute path you bound it to as the --config flag. For example with `docker run` you can just use `-v`.

**Reply by Techal62** [reply]
Thank you very much, it works very well.
I lack knowledge on how docker works. I will study this more deeply.
